.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,97 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+# ******** NOTE ********
+
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master, beta*, release-* ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '28 9 * * 6'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: windows-2019
+    
+    env:
+      Configuration: Release
+      SolutionFile: dotnet\DotNetStandardClasses.sln
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'csharp' ]
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python' ]
+        # Learn more...
+        # https://docs.github.com/en/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#overriding-automatic-language-detection
+
+    steps:
+      # Fixes issue of JVM running out of memory when analyzing the scan results
+    - name: configure Pagefile
+      uses: al-cheb/configure-pagefile-action@7e234852c937eea04d6ee627c599fb24a5bfffee # SHA of v1.2
```

Comment:
Consider update to v1.3 (https://github.com/al-cheb/configure-pagefile-action/releases/tag/v1.3)

Reply:
Removed. It causes a memory error on GitHub runners with 7GB of RAM.